src/test/java/com/karankumar/bookproject/ui/book/BookFormTest.java

```diff
@@ -347,7 +347,7 @@ void correctFormFieldsShowForDidNotFinishShelf() {
 
     @ParameterizedTest
     @MethodSource("shelfNames")
-    void fieldsToResetAreCorrectlyPopulated(
+    void populateCorrectlyFieldsToReset(
```

Comment:
Could you please move 'Correctly' to the end (so we get 'populateFieldsToResetCorrectly')?